README.md

```diff
@@ -1 +1,72 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```

Comment:
- [ ] Kindly replace the `Project Name` with your actual name like `Hello Microverse`.